platformio.ini

```diff
@@ -760,7 +763,8 @@ monitor_speed     = 115200
 platform          = ${common_stm32f1.platform}
 extends           = common_stm32f1
 board             = MEEB_3DP
-platform_packages = tool-stm32duino
+platform_packages = ${common_stm32f1.platform_packages}
+  tool-stm32duino
```

Comment:
Does anybody know if it would harm anything to add this dependency to _all_ STM32F1 environments? I don't know what this tool is actually used for, but it would be nice if we could move it up to `common_stm32f1` and delete all these.

Reply:
stm32flash (part of tool-stm32duino) is for upload the firmware to boards like Fysect AIOII board (for example) (env:STM32F103RC_fysetc), because comes without bootloader, and the unique method to upload the firmware is that tool

Reply:
Okay thanks, I think it should be fine to move it to `common_stm32f1` for simplicity then.